src/Illuminate/Database/Connectors/MySqlConnector.php

```diff
@@ -175,6 +175,6 @@ protected function setCustomModes(PDO $connection, array $config)
      */
     protected function strictMode()
     {
-        return "set session sql_mode='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION'";
+        return "set session sql_mode='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION'";
```

Comment:
These are 2 changes in 1 commit (& PR), split it up for discussion

Reply:
This PR:
https://github.com/laravel/framework/pull/23948
also is about changing strict mode